advocacy_docs/edb-postgres-ai/ai-accelerator/retrievers/usage.mdx

```diff
@@ -109,11 +109,11 @@ SELECT aidb.create_retriever_for_volume(
        );
 ```
 
-In this example, we use all the defaults.
+This example uses all the defaults.
```

Comment:
I was a little unclear on this wording. Are you saying that the example uses default values for all the settings? Or the settings that aren't otherwise specified? A little curious why this needs to be mentioned.